llvm/test/Transforms/LoopVectorize/last-min-index.ll

```diff
@@ -0,0 +1,84 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -passes=loop-vectorize -S %s | FileCheck %s --check-prefix=CHECK-REV-MIN
```

Comment:
This will need something like `-force-vector-width=4` to make sure this gets vectorized if possible.
```suggestion
; RUN: opt -passes=loop-vectorize -force-vector-width=4 -S %s | FileCheck %s
```